graphqlapi/readme.md

```diff
@@ -0,0 +1,24 @@
+# GraphQL endpoint generation for Weaviate
+The point of external access is in `graphql_schema.go` (`CreateSchema()`). The schema is assembled in its entirety in `assembleFullSchema()` in `build_schema.go`.
+
+## The schema
+The schema is based on `GraphQLObjects`. These `GraphQLObjects` have a property that contains a `GraphQL.Fields{}` object, which is a hashmap. This map contains one or more `GraphQL.Field` objects. These `Field` objects have `Scalar` values or other `GraphQLObjects` as value. 
```

Comment:
* `GraphQL.Fields{}` -> `graphql.Fields` (the {} refers to instantiating a literal map, see [ref docs](https://blog.golang.org/go-maps-in-action), search for 'To initialize a map with some data, use a map literal: ')
* which is a hashmap -> which is a `map[string]graphql.Field`